recipes/expressbetadiversity/build.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# Install package into the share directory to allow for unit tests to be run.
+SHARE_DIR=${PREFIX}/share/${PKG_NAME}-${PKG_VERSION}-${PKG_BUILDNUM}
+mkdir -p ${SHARE_DIR}
+
+# Copy across unit tests and python scripts
+cp -r unit-tests ${SHARE_DIR}
+
+# Build the binary
+cd source
+ln -s ${CC} $BUILD_PREFIX/bin/gcc
+ln -s ${CXX} $BUILD_PREFIX/bin/g++
+make
+
+# Move built binary across to the shared bin and symlink it
+mkdir -p ${SHARE_DIR}/bin/
+cd ../bin
+chmod +x ExpressBetaDiversity
+cp ExpressBetaDiversity ${SHARE_DIR}/bin/
+
+# Create the binary
+mkdir -p ${PREFIX}/bin/
+cat <<EOF >${PREFIX}/bin/ExpressBetaDiversity
+#!/bin/bash
+cd ${SHARE_DIR}/bin/
```

Comment:
Can you add a comment explaining why you need the `cd`?

Reply:
Added some comment, but this is mainly due to the unit tests being called relative to the binary location (../unit-tests). Alternatively, a patch file could be created before building it but I'm not entirely sure if that is the cleaner solution? I'm open to give it a go if that is what bioconda prefers.

Reply:
Sorry I haven’t looked too closely now, but what I think I don’t understand is why the installed version of the package needs to be modified for unit tests, while I would expect unit tests to be run only at build time.

Reply:
As would I, but there is an argument that can be passed (-u) when running the program which runs all unit tests. So this is being kept purely for that functionality.

Reply:
Are there otherwise ever file paths given to this? My primary concern is that the `cd` will cause issues if relative paths are provided (I make copious use of them when analyzing data). If that's not the case, or there's something that I don't understand that makes that not break then I'm fine with this. Otherwise I would suggest that breaking unit tests is a fine compromise.